hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OmTableInsightTask.java

```diff
@@ -287,18 +344,30 @@ private void handlePutEvent(OMDBUpdateEvent<String, Object> event,
 
   private void handleSizeRelatedTablePutEvent(
       OMDBUpdateEvent<String, Object> event, String countKey,
-      String unReplicatedSizeKey, String replicatedSizeKey,
+      String unReplicatedSizeKey, String replicatedSizeKey, String tableName,
       HashMap<String, Long> objectCountMap,
       HashMap<String, Long> unreplicatedSizeCountMap,
-      HashMap<String, Long> replicatedSizeCountMap) {
+      HashMap<String, Long> replicatedSizeCountMap) throws IOException {
+
+    boolean isFileTable = tableName.equals(OPEN_FILE_TABLE);
```

Comment:
variable name is confusing as these are for openfile and openkey tables, though these might not be needed if we can control at the caller of this method itself.